### DISCUSSION TITLE
Supabase error "control reached end of trigger procedure without RETURN" when updating data

### DISCUSSION THREAD

**Discussion by PriyavKaneria** [supabase/supabase#4441] [Questions] [answered]
Hello guys!
I am using supabase for backend and svelte for frontend
When I try and update a table `vehicles_make` using the following code :
![image](https://user-images.githubusercontent.com/43821039/145685735-876bddb1-1a19-428a-af3b-314fa204688f.png)
Here vehicleMakeData is `{is_active:false}` and id is a valid integer id present in db.
The following error comes up :
```json
{
    message: 'control reached end of trigger procedure without RETURN',
    code: '2F005',
    hint: null,
    details: null
}
```
This error only comes when {is_active:false} is passed to update and works fine when {is_active:true} is used
Also all other tables are updating the same value without any issue
If anyone can help it would be great!

**Comment by GaryAustin1** [comment, marked as answer]
Do you have a trigger function on that table for updates?  Pretty sure that is saying it is not returning data to the update operation.

**Reply by PriyavKaneria** [reply]
Thank you for quick response 😇

**Comment by krishnardt** [comment]
Yes @GaryAustin1 , we found that the issue is only because the trigger is executing without returning anything.
After adding return statement, it worked like a charm. Thanks for the suggestion.
@PriyavKaneria We can close this issue now.
